src/resources/users/user-model.js

```diff
@@ -60,10 +59,23 @@ async function add(user) {
   return findById(id);
 }
 
-function update(id, body) {
-  return db('users')
+// async function update(id, body) {
+//   const userUpdate = await db('users')
+//     .where({ id })
+//     .update(body);
+
+//   return findById(id);
+// }
```

Comment:
Here is also a comment. I saw you have merge conflicts, please pull from the Dev branch and resolve them. Everything else seems fine, I will merge it after you made the changes. Great job Ola 👍👍👍

Reply:
merge conflict resolved